### PR TITLE
Added icons as labels for inputs

### PR DESCRIPTION
**Pull request by carlosascari**
upload image avatar is now first option unless an social avatar is fetched
fixed ImagePicker badge bug where badge images would not resize to fit

For issue https://github.com/OpenCollective/OpenCollective/issues/103

### CODE REVIEW

File: frontend/src/components/ImagePicker.js [hunk 1]
```diff
@@ -197,35 +197,45 @@ export default class ImagePicker extends Component {
...
+    if (!this.state.isLoading) {
       this.setState({isLoading: true});
     }
 
     getSocialMediaAvatars(newUser.id, {website: website, twitterHandle: twitter, name: profileForm.name})
     .then((response) => {
       const currentOption = this.options[this.state.currentIndex];
       const results = response.json.filter((option) => this.blacklist.indexOf(option.src) === -1 );
+      const newResults = results.filter((option) => {
+        let alreadyExists = false;
```
Comment (asood123) [inline frontend/src/components/ImagePicker.js]: Is there a benefit of using `let` vs `var`? If not, could you change to `var`? It's used everywhere else in the codebase.

File: frontend/src/components/ImagePicker.js [hunk 2]
```diff
@@ -197,35 +197,45 @@ export default class ImagePicker extends Component {
...
+        this.options.forEach((opt) => {
+          if (opt.src === option.src) {
+            alreadyExists = true;
+          }
+        })
+        return !alreadyExists
+      });
       let isFirstTime = false;
```
Comment (asood123) [inline frontend/src/components/ImagePicker.js]: same as above.

**Comment by asood123** [comment]
Besides the `let` issue, everything else looks good

File: frontend/src/components/ImagePicker.js (hunk 1 above)
Comment (carlosascari) [inline frontend/src/components/ImagePicker.js]: @asood123 
I've noticed `var` used on the server side, I've seen `let` used throughout the frontend. In ES6 `let` is favoured over `var`, and they do differ http://stackoverflow.com/a/11444416

Comment (sedubois) [inline frontend/src/components/ImagePicker.js]: I prefer the `let` as well, the narrower the scoping the better. (I come from the Java world...)

**Comment by asood123** [comment]
`let` makes more sense. I added it to our linter wishlist https://github.com/OpenCollective/OpenCollective/issues/76

**Pull request merged**